<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background: rgb(200, 0, 45);
opacity: 3;
border-radius: 10px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    border: 2px solid #fff; /* Белая рамка */</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="blur">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;
    border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>        blur</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_blur">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>21</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_blur">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="intValue" stdset="0">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>161</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    border: 2px solid #fff; /* Белая рамка */</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="monochrome">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;
    border-radius: 10px;</string>
     </property>
     <property name="text">
      <string> monochrome</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>161</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    border: 2px solid #fff; /* Белая рамка */</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="sharpen">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;
    border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>     sharpen</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_sharpen">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>21</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_sharpen">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="intValue" stdset="0">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="accept_btn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
background: rgb(200, 0, 45);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <zorder>frame_4</zorder>
   <zorder>frame_3</zorder>
   <zorder>frame_2</zorder>
   <zorder>accept_btn</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
